superset/commands/importers/v1/utils.py

```diff
@@ -87,3 +88,11 @@ def is_valid_config(file_name: str) -> bool:
         return False
 
     return True
+
+
+def get_contents_from_bundle(bundle: ZipFile) -> Dict[str, str]:
```

Comment:
nit: write small unit test to cover this

Reply:
This is covered in the API tests